src/map_io/map_flagdata_packet.cc

```diff
@@ -65,7 +65,9 @@ void MapFlagdataPacket::read(FileSystem& fs,
 
 					//  Owner is already set, nothing to do from PlayerImmovable.
 
-					flag.animstart_ = fr.unsigned_16();
+					// TODO(Nordfriese): Change this line when we next increase the packet version:
```

Comment:
Please add "saveloading" and/or "Compatibility" to these comments so we can do a fulltext search on them later. I would definitely miss those otherwise.